lib/bolt/inventory/inventory2.rb

```diff
@@ -196,8 +196,7 @@ def resolve_name(target)
       private :resolve_name
 
       def expand_targets(targets)
-        if targets.is_a? Bolt::Target
-          targets.inventory = self
+        if targets.is_a? Bolt::Target2
```

Comment:
`kind_of(Bolt::Target)` is probably safer.

Reply:
I can dig in to the differences. Rubocop prefers `is_a?`
```
lib/bolt/inventory/inventory2.rb:197:20: C: Style/ClassCheck: Prefer Object#is_a? over Object#kind_of?.
        if targets.kind_of?(Bolt::Target2)
```

Reply:
There is no difference.